File: Chapter0_Algorithm/2402/D240208/bj_5086.java
package D240208;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class bj_5086 {
    public static void main(String[] args) throws NumberFormatException, IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String result = "";
        while (true) {
            String[] n = br.readLine().split(" ");
            int a = Integer.parseInt(n[0]);
            int b = Integer.parseInt(n[1]);

            if (a == 0 && b == 0)
                break;
            else if (b % a == 0)
                // System.out.println("factor");
                result += "factor";
            else if (a % b == 0)
                // System.out.println("multiple");
                result += "multiple";
            else
                // System.out.println("neither");
                result += "neither";

            result += "\n";
        }
        System.out.println(result.substring(0, result.length() - 1));

    }
}

File: Chapter0_Algorithm/2402/D240208/bj_5086.py

a = 1
b = 1
result = []
while(1) :
    a, b = list(map(int, input().split(" ")))
    if a==0 and b==0 :
        break
    elif b%a == 0 :
        result.append("factor")
    elif a%b == 0 :
        result.append("multiple")
    else :
        result.append("neither")

print("\n".join(result))

